src/Microsoft.AspNetCore.OData/Extensions/ODataServiceCollectionExtensions.cs

```diff
@@ -113,6 +113,33 @@ public static IODataBuilder AddOData(this IServiceCollection services)
             return new ODataBuilder(services);
         }
 
+        /// <summary>
+        /// Enables case-insensitive model binding.
+        /// </summary>
+        /// <param name="builder">Odata Builder.</param>
+        public static IODataBuilder EnableODataCaseInsensitiveModelBinding(this IODataBuilder builder)
```

Comment:
>IODataBuilder [](start = 80, length = 13)

It should be "IServiceCollection" ,because the setting is global, not for the certain OData route?

Reply:
@xuzhg I initially used `IServiceCollection`. You asked me to use `IODataBuilder` in this comment https://github.com/OData/WebApi/pull/2249#discussion_r484522022

Reply:
> @xuzhg I initially used `IServiceCollection`. You asked me to use `IODataBuilder` in this comment [#2249 (comment)](https://github.com/OData/WebApi/pull/2249#discussion_r484522022)

Oh, yes.

Reply:
I will retain `IODataBuilder` so that the customers will be able to use the setting as follows.
`services.AddOData().EnableODataCaseInsensitiveModelBinding(true);`

#Resolved